### ISSUE TITLE
Doesn't sync with nextcloud android app

### ISSUE THREAD

**Issue by xorinzor** [nextcloud/contacts#2031]
**Is your feature request related to a problem? Please describe.**
I expected when I uploaded my contacts via the Nextcloud mobile app for them to appear in the nextcloud contacts list.
Instead, all it seems to do is create a .vcf file which then has to be manually imported (and which doesn't check for duplicates either, so you're required to delete all of them, before importing them again).

**Describe the solution you'd like**
In a nutshell, better synchronisation. Detect when contacts are duplicates and (offer to) merge them. Not only that, but also (if at all possible) utilize the existing contact upload functionality from the app to tightly integrate it into this app.

**Describe alternatives you've considered**
For now, sticking with the stock android / google solution for contact synchronisation.

**Comment by jakobroehrl** [comment]
You need the Davx5 App to sync contacts with Android
https://f-droid.org/de/packages/at.bitfire.davdroid/

**Comment by skjnldsv** [comment]
See above
